src/json/from_json.cr

```diff
@@ -58,6 +58,14 @@ def Array.from_json(string_or_io) : Nil
   nil
 end
 
+def Deque.from_json(string_or_io) : Nil
+  parser = JSON::PullParser.new(string_or_io)
+  new(parser) do |element|
+    yield element
+  end
+  nil
```

Comment:
Redundant

```suggestion
```

Reply:
just copied the Array.from_json implementation